src/components/MainTable/MainTable.js

```diff
@@ -219,6 +229,10 @@ const MainTable = ({
 MainTable.propTypes = {
   defaultSort: PropTypes.string,
   defaultSortDirection: PropTypes.oneOf(["ascending", "descending"]),
+  /**
+   * A state that will be shown when no rows are passed to the table.
+   */
+  emptyState: PropTypes.node,
```

Comment:
You're not displaying this node if it is provided.

Reply:
See line 89 

<img width="788" alt="Screenshot 2020-01-13 at 14 19 56" src="https://user-images.githubusercontent.com/505570/72262949-d4c04080-360f-11ea-8ae9-29f767fc83b7.png">